proctor/src/main/java/TDS/Proctor/Services/remote/RemoteTestOpportunityService.java

```
@@ -174,7 +180,14 @@ public boolean approveOpportunity(final UUID examId, final UUID sessionId, final
             return isApproveSuccessful;
         }
 
-        examRepository.updateStatus(examId, STATUS_APPROVED, IN_USE.getType(), null);
+        Optional<ValidationError> maybeError = examRepository.updateStatus(examId, STATUS_APPROVED, IN_USE.getType(), null);
+
+        //Consistent with the legacy implementation.  If it fails for any reason it throws.
+        if(maybeError.isPresent()) {
```

We were swallowing any error that happened when approving or denying students.